pkg/sdn/plugin/egress_network_policy.go

```diff
@@ -13,6 +14,10 @@ import (
 )
 
 func (plugin *OsdnNode) SetupEgressNetworkPolicy() error {
+	if err := CheckDNSResolver(); err != nil {
+		return err
```

Comment:
You should just remove this from this commit rather than adding it here and then removing it in a later commit. (Likewise in proxy.go)